src/differential_privacy/aggregates.rs

```diff
@@ -33,9 +30,19 @@ impl Field {
 
 impl Relation {
     fn gaussian_mechanism(self, epsilon: f64, delta: f64, bounds: Vec<(&str, f64)>) -> DPRelation {
```

Comment:
private_query::gaussian_noise should be private_query::gaussian_mechanism
All mechanisms should be in the same module defined as close as possible to the textbook

rename this to gaussian_mechanism**s**